pkg/charts/util.go

```diff
@@ -0,0 +1,94 @@
+package charts
```

Comment:
naming nit: maybe just call this file `charts.go`? prefer to avoid `util` where possible.